content/docs/get-started/applying.md

```diff
@@ -12,17 +12,24 @@ Now, we can use MLEM to load the model and calculate some metrics:
 # predict.py
 from mlem.api import load
 
-
 model = load("models/rf")
-y_pred = model.predict_proba([[0, 1, 2, 3]])
+features = [
+    "sepal length (cm)",
+    "sepal width (cm)",
+    "petal length (cm)",
+    "petal width (cm)",
+]
+x = pd.DataFrame([[0, 1, 2, 3]], columns=features)
+y_pred = model.predict_proba(x)
```

Comment:
Why? Using a simple list is more concise. This should demonstrate the value, and using list does the job.

Reply:
Take a look at the first issue here: https://github.com/iterative/mlem.ai/issues/213

Reply:
Not sure which is better TBH. @jorgeorpinel, WDYT?

Reply:
Both arguments are compelling so it's a tradeoff. Considering that many readers don't actually try the commands I'd err on the side of short and concise. Hopefully people who try it will just ignore the warning.

Reply:
@aguschin @jorgeorpinel 
Sorry but here I have to push back - if someone running the getting started is hitting a user warning it looks very bad. especially if it's literally on the first command they hit in this guide. I don't think we should accept this.
I don't mind taking the other route mentioned [here](https://github.com/iterative/mlem.ai/issues/213#issuecomment-1304819752) - removing the column names form the training piece of code and the batch-scoring example. it will make the whole thing shorter. Is it prettier or becomes a bit meaningless 🤷 arguable - but I think either way (removing features on training as well OR adding them in this snippet) is better than getting a runtime userWarning 😢

Reply:
I don't see it as a big deal. Also, it's not a MLEM warning. Use `python predict.py 2>/dev/null` ? Up to you, guys.

Reply:
ok, let's keep the suggestion. Doing it the other way (removing column names) is even worse, since `.mlem` file will be much less informative - that just hurts the other part of GS.

Reply:
Thanks @aguschin 
Sorry for pushing on this folks, but I feel strongly about not punishing/disappointing people trying the commands in our doc guides 😓 if creates a bad impression imo